actions/get-vault-secrets/README.md

```diff
@@ -44,3 +46,46 @@ jobs:
 ```
 
 <!-- x-release-please-end-version -->
+
+### Using Outputs
+
+You can also use the action with `export_env: false` to get secrets as outputs instead of environment variables:
+
+```yaml
```

Comment:
Does this need to be within `x-release-please-start-version` like the example above?

Reply:
Good catch! Added.